app/models/repository.rb

```diff
@@ -1,3 +1,7 @@
 class Repository < KalibroClient::Entities::Processor::Repository
   include KalibroRecord
+
+  def self.latest(count=1)
+    all.sort { |one, another| another.id <=> one.id }.first(count)
```

Comment:
This is not activerecord, but KalibroClient... The `all` will retrieve an array once per call and the sort will produce no other calls as it is implemented by array.

Do you have a better implementation?